aten/src/ATen/native/group_norm.cpp

```diff
@@ -80,7 +80,7 @@ std::tuple<Tensor, Tensor, Tensor> native_group_norm(
   auto memory_format = X.device().is_cpu() ?
       X.suggest_memory_format() : at::MemoryFormat::Contiguous;
 
-  TORCH_CHECK(X.is_contiguous(memory_format));
+  TORCH_CHECK(X.is_contiguous(memory_format) || (X.device().is_xpu() && memory_format == at::MemoryFormat::Contiguous));
```

Comment:
`X.device().is_xpu() && memory_format == at::MemoryFormat::Contiguous` is always `True`. Because `X.device().is_cpu()` is `False` and then `auto memory_format = at::MemoryFormat::Contiuous`. So it means the statement is always True if `X` is an Intel GPU tensor. Is that expected?

Reply:
Yeah,  I change the code to make sure we have contiguous input&output always, this should be more proper than the above codes.